### ISSUE TITLE
Tensorflow support for model compression

### ISSUE THREAD

**Issue by aqibsaeed** [microsoft/nni#4350]
Hi,

Is there a support/API for model compression of Tensorflow models? Thanks.

**Comment by liuzhe-lz** [comment]
There is a very basic pruning support for TensorFlow. ([example](https://github.com/microsoft/nni/blob/master/examples/model_compress/pruning/naive_prune_tf.py))
It only generates masks and does not support actual speed-up.

TensorFlow support is in the plan list but we are short of developers. PyTorch support is way much more mature.

**Comment by aqibsaeed** [comment]
Thank you for the quick response. The example demonstrate during training pruning, I am looking for something even simpler that is post-training pruning based on L1-Norm. Is there a support for that?
